airbyte-integrations/connectors/source-google-search-console/source_google_search_console/source.py

```diff
@@ -182,7 +187,7 @@ def get_custom_reports(self, config: Mapping[str, Any], stream_config: Mapping[s
     def get_stream_kwargs(self, config: Mapping[str, Any]) -> Mapping[str, Any]:
         return {
             "site_urls": config["site_urls"],
-            "start_date": config["start_date"],
+            "start_date": config.get("start_date") or "2021-01-01",  # `2021-01-01` is the default value
```

Comment:
Why do you choose to use the `or` operator instead of utilizing the default value option with the `get` method?

```suggestion
            "start_date": config.get("start_date", "2021-01-01"),  # `2021-01-01` is the default value
```

Reply:
there are some unit tests checking the presence of the `start date` in the config, and checking the `""` as start date value, if we run local tests the start date is not set yet, because the spec isn't handled yet by the ui, so `default` spec value for the `start date` is not set yet. To overcome this we have to add the default value in two places: spec and code, `or` is used to overcome the empty string scenario.

Reply:
done)